app/models/permission.rb

```diff
@@ -0,0 +1,29 @@
+class Permission
+  def initialize(user)
+    if user.nil?
+      allow 'home', [:index]
+      allow 'sessions', [:create]
+      allow 'api/version1_controller', [:get_proxy, :report_result]
```

Comment:
Yeah, I saw that.  If we're going to add AA to the web UI I feel we should add it to the API as well.